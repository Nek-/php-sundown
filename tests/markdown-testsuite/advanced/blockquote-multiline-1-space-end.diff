002+ <p>A blockquote on multiple lines like this.&Acirc;&nbsp;</p>
002- <p>A blockquote on multiple lines like this.</p>