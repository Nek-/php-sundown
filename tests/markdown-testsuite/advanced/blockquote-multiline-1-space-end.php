<?php
$data = <<< DATA
>A blockquote 
>on multiple lines 
>like this.Â 
DATA;
$md = new Sundown\Markdown(new Sundown\Render\Html());
$result = $md->render($data);

$tidy = new tidy;
$tidy->parseString($result, array("show-body-only"=>1));
$tidy->cleanRepair();
echo (string)$tidy;
