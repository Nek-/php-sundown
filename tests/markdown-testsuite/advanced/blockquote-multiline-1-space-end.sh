#!/bin/sh

/Users/chobie/.phpenv/versions/5.5.6-debug-zts/bin/php  -n -c '/Users/chobie/src/php/ext/php-sundown/tmp-php.ini'  -d "output_handler=" -d "open_basedir=" -d "safe_mode=0" -d "disable_functions=" -d "output_buffering=Off" -d "error_reporting=32767" -d "display_errors=1" -d "display_startup_errors=1" -d "log_errors=0" -d "html_errors=0" -d "track_errors=1" -d "report_memleaks=1" -d "report_zend_debug=0" -d "docref_root=" -d "docref_ext=.html" -d "error_prepend_string=" -d "error_append_string=" -d "auto_prepend_file=" -d "auto_append_file=" -d "magic_quotes_runtime=0" -d "ignore_repeated_errors=0" -d "precision=14" -d "memory_limit=128M" -d "opcache.fast_shutdown=0" -d "extension_dir=/Users/chobie/src/php/ext/php-sundown/modules/" -d "extension=sundown.so" -d "session.auto_start=0" -d "tidy.clean_output=0" -d "zlib.output_compression=Off" -d "mbstring.func_overload=0" -f "/Users/chobie/src/php/ext/php-sundown/tests/markdown-testsuite/advanced/blockquote-multiline-1-space-end.php"  2>&1
