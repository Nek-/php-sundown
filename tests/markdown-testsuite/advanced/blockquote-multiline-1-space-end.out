<blockquote>
<p>A blockquote on multiple lines like this.&Acirc;&nbsp;</p>
</blockquote>